crates/nu-command/src/filters/last.rs

```diff
@@ -20,6 +20,7 @@ impl Command for Last {
     fn signature(&self) -> Signature {
         Signature::build("last")
             .input_output_types(vec![
+                (Type::List(Box::new(Type::String)), Type::String),
```

Comment:
Actually you don't need to add this

Reply:
reverted, we'll see if the CI is happy :crossed_fingers: